### ISSUE TITLE
Cursor position is wrong.

### ISSUE THREAD

**Issue by lyuha** [careteditor/caret#282]
<img width="661" alt="workspace log blog _posts 2017-03-31-move-pyvenv md 2017-03-31 04-59-46" src="https://cloud.githubusercontent.com/assets/4014016/24523769/29142480-15cf-11e7-8c82-ca610e3077b9.png">

Above seen, cursor position is so far from expected position. I test with CJK sentence, this problem occurs with only korean characters.

**Comment by astoilkov** [comment]
@lyuha Can you paste the line of text that you are having problems with? Can you also share what font you are using?

Thanks in advance.

**Comment by lyuha** [comment]
@astoilkov Any sentenes using korean. Below, generated paragraphs have same issue.

```
것이다. 뜨거운지라, 얼음이 하는 우리 주며. 든 뿐이다, 청춘 아름답고 수 있는 찬미를 피고 그것은 방지하는 아니다, 위하여서 석가는 광야에서 지혜는 것이다, 그들의 얼마나 반짝이는 많이 평화스러운 우리의 같은 칼이다. 보이는 얼마나 뛰노는 얼음과 피가 사막이다.
눈이 동력은 그들의 이상. 맺어. 살았으며. 별과 그것을 가는 피다, 두손을 우는 충분히 뿐이다. 열락의 천자만홍이 지혜는 바이며. 얼음에 그들의 시들어 하였으며, 얼음에 따뜻한 사는가 이것이다.
```

I use default preference.

```yaml
font: Roboto
fontSize: 15
lineHeight: 1.9
lineWidth: 72
```

**Comment by astoilkov** [comment]
One last thing. What version of Caret are you using? There was such an issue in between 2.0.0 and 2.0.3. Maybe your app haven't been updated yet.

**Comment by lyuha** [comment]
latest version. v2.0.5.

**Comment by astoilkov** [comment]
Thanks a lot for the information. We found the problem. Expect fix soon.

**Comment by astoilkov** [comment]
We fixed the problem in Caret 2.0.6. Can you verify that?

**Comment by lyuha** [comment]
Yes. This problem was fixed.
